src/software/time/time.cpp

```diff
@@ -23,3 +20,27 @@ double Time::getMilliseconds() const
 }
 
 Time::~Time() {}
+
+double Time::now()
+{
+    const auto clock_now = std::chrono::system_clock::now();
```

Comment:
Hmm, I'm not sure this should be a part of this class. Ideally we prevent anyone inside our AI system from ever checking the system time, we should be doing operations based off timestamps. The only places (IMO) where we should ever even look at the system time is outside AI (and even then only in limited cases), most stuff should be based off the timestamps from the raw messages.

Reply:
this is used for create TbotsSoftwareMsg

Reply:
after looking at it again, it's probably not gonna be used enough to be its own static function, so I reversed this change